tools/build/configure.py

```diff
@@ -114,16 +114,37 @@ def write_config():
                     bazel_rc.write('build --action_env TF_CUDA_VERSION="10.1"\n')
                     bazel_rc.write('build --action_env TF_CUDNN_VERSION="7"\n')
             # Needed for tf rules
-            bazel_rc.write('build --experimental_repo_remote_exec\n')
+            bazel_rc.write("build --experimental_repo_remote_exec\n")
             # Enable platform specific config
-            bazel_rc.write('build --enable_platform_specific_config\n')
+            bazel_rc.write("build --enable_platform_specific_config\n")
             # Needed for GRPC build
             bazel_rc.write('build:macos --copt="-DGRPC_BAZEL_BUILD"\n')
             # Stay with 10.14 for macOS
             bazel_rc.write('build:macos --copt="-mmacosx-version-min=10.14"\n')
             bazel_rc.write('build:macos --linkopt="-mmacosx-version-min=10.14"\n')
             # MSVC (Windows): Standards-conformant preprocessor mode
             bazel_rc.write('build:windows --copt="/Zc:preprocessor"\n')
+            # Config for CI and release build
+            bazel_rc.write("build:ci --copt=-msse4.2\n")
+            bazel_rc.write("build:ci --copt=-mavx\n")
+            bazel_rc.write("build:ci --compilation_mode=opt\n")
+            bazel_rc.write("build:mac_ci --copt=-Wunguarded-availability\n")
+            bazel_rc.write(
+                "build:linux_ci --crosstool_top=//third_party/toolchains/gcc7_manylinux2010:toolchain\n"
+            )
+            bazel_rc.write(
+                "build:linux_ci_gpu --crosstool_top=//third_party/toolchains/gcc7_manylinux2010-nvcc-cuda10.1:toolchain\n"
+            )
+            bazel_rc.write("build:output_ci --noshow_progress\n")
+            bazel_rc.write("build:output_ci --noshow_loading_progress\n")
+            bazel_rc.write("build:output_ci --verbose_failures\n")
+            bazel_rc.write("build:output_ci --test_output=errors\n")
```

Comment:
Maybe we can fold all into part of the normal build, as they are not impactful on the binary output even for normal build?
```
build --noshow_progress
build --noshow_loading_progress
build --verbose_failures
build --test_output=errors
```

We can also add `--experimental_ui_max_stdouterr_bytes=-1` here as well:
```
build --experimental_ui_max_stdouterr_bytes=-1
```

Reply:
Done !